apps/website/src/pages/api/courses/[courseSlug]/group-switching/available.ts

```diff
@@ -89,45 +85,40 @@ function calculateGroupAvailability({
       discussionsByUnit[unitKey] = [];
     }
 
+    const hasStarted = discussion.startDateTime * 1000 <= now;
+
     discussionsByUnit[unitKey].push({
       discussion,
       spotsLeft,
```

Comment:
For groups you've made `spotsLeft` depend on whether `hasStarted` is true or not. Do we not need to make that distinction for discussions?